src/main/java/com/force/i18n/grammar/NounForm.java

```diff
@@ -33,11 +33,4 @@ public interface NounForm extends GrammaticalForm {
      * @return the possessive associated with this noun form
      */
     LanguagePossessive getPossessive();
-
-    /**
-     * @return a HTML compatible screen that can be used to represent this noun
-     * form uniquely when compared to all other noun forms.
-     */
-    @Override
-    String getKey();
```

Comment:
this is redundant. the super class (interface) `GrammaticalForm` already has the same method signature